### ISSUE TITLE
Consider Using doctest-modules

### ISSUE THREAD

**Issue by akaszynski** [pyansys/DPF-Core#62]
We test all docstrings using a pytest function in:
https://github.com/pyansys/DPF-Core/blob/a32b3ca0c18a32376c98ab630977be1d01318e48/tests/test_code_docstrings.py#L27-L46

Is there any reason why we can't use doctest-modules?  For example:
```
pytest -v --doctest-modules
```

**Comment by cbellot000** [comment]
Good point @akaszynski, I guess we didn't know that it existed, @anslpa wrote this unit test actually. Léa, I think that we should remove this unit test then and use `pytest -v --doctest-modules` in a new task of our pipelines. I just tried it locally and it's a lot cleaner since pytest gives one test result by docstring and behave just like a regular test
